sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1.java

```diff
@@ -1152,7 +1161,14 @@ public void startBundle(StartBundleContext c) {
 
     @ProcessElement
     public void processElement(ProcessContext c) throws Exception {
```

Comment:
Deletes are fine, I think, as they wouldn't send enough data per entity to trouble this limit.

Let's discuss docs separately.